src/main/java/ai/deepcode/javaclient/responses/Suggestion.java

```diff
@@ -16,20 +16,21 @@
   private final List<String> categories;
   private final List<String> tags;
   private final List<String> cwe;
+  private final String leadURL;
 
   public Suggestion(
-      String id,
-      String rule,
-      String message,
-      String title,
-      String text,
-      int severity,
-      int repoDatasetSize,
-      List<String> exampleCommitDescriptions,
-      List<ExampleCommitFix> exampleCommitFixes,
-      List<String> categories,
-      List<String> tags,
-      List<String> cwe) {
+          String id,
+          String rule,
+          String message,
+          String title,
+          String text,
+          int severity,
+          int repoDatasetSize,
+          List<String> exampleCommitDescriptions,
+          List<ExampleCommitFix> exampleCommitFixes,
+          List<String> categories,
+          List<String> tags,
+          List<String> cwe, String leadURL) {
```

Comment:
doesn't linter complain? feel free to ignore the comment 👍

Reply:
reformatted